chef/cookbooks/hyperv/recipes/register_services_neutron.rb

```diff
@@ -16,5 +16,5 @@
 service "neutron-hyperv-agent" do
   service_name node[:service][:neutron][:name]
   action [:enable, :start]
-  subscribes :restart, "template[#{node[:openstack][:config].gsub(/\\/, "/")}/neutron_hyperv_agent.conf]"
+  subscribes :restart, "template[#{node[:openstack][:neutron][:config].gsub(/\\/, "/")}/neutron_hyperv_agent.conf]"
```

Comment:
Metrics/LineLength: Line is too long. [115/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)